CONTRIBUTING.md

```diff
@@ -124,7 +124,7 @@ and this will be downloaded on `make start` to be injected into the VMs by ansib
 
 Unfortunately, as of this writing, due to security concerns we do not publish
```

Comment:
this whole paragraph can be removed, actually. we publish at github.com/contiv/build now.